### PR TITLE
add a lot of missing localization

### PR DESCRIPTION
**Pull request by TobiTenno**
**Summary:**
add a lot of missing localization

---
**Fixes issue (include link):**

---
**Mockups, screenshots, evidence:**

### CODE REVIEW

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5335813&bid=17466094)

File: src/components/panels/BountyPanel.vue [hunk 1]
```diff
@@ -67,7 +67,7 @@ export default {
   props: ['syndicate', 'type'],
   computed: {
     headertext() {
-      return `${this.type} ${this.$t('bounty.header')}`;
+      return this.$t('bounty.header', {type: this.$t(`timer.${this.type.toLowerCase().replace(/\s/gi, '-')}`)});
```
Comment (EricSihaoLin) [inline src/components/panels/BountyPanel.vue:70]: Codacy Issue found: [Replace `type:·this.$t(`timer.${this.type.toLowerCase().replace(/\s/gi,·'-')}`)` with `·type:·this.$t(`timer.${this.type.toLowerCase().replace(/\s/gi,·'-')}`)·`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5335813)

File: src/components/panels/EventsPanel.vue [hunk 2]
```diff
@@ -22,14 +22,14 @@
           <b-badge variant="danger">{{ event.victimNode }}</b-badge>
         </div>
         <b-badge v-if="isHealthReversed(event)" class="event-health" :variant="eventHealthVariantOpposite(event)">
-          {{ event.health || (100 - (event.currentScore / event.maximumScore) * 100).toFixed(2) }}% Completed
+          {{ $t('event.completed', {perc: event.health || (100 - (event.currentScore / event.maximumScore) * 100).toFixed(2) }) }}
         </b-badge>
         <b-badge v-else class="event-health" :variant="eventHealthVariant(event)">
-          {{ event.health || (100 - (event.currentScore / event.maximumScore) * 100).toFixed(2) }}% Remaining
+          {{ $t('event.remaining', {perc: event.health || (100 - (event.currentScore / event.maximumScore) * 100).toFixed(2) }) }}
```
Comment (EricSihaoLin) [inline src/components/panels/EventsPanel.vue:28]: Codacy Issue found: [Replace `·$t('event.remaining',·{perc:·event.health·||·(100·-·(event.currentScore·/·event.maximumScore)·*·100).toFixed(2)·})` with `⏎············$t('event.remaining',·{⏎··············perc:·event.health·||·(100·-·(event.currentScore·/·event.maximumScore)·*·100).toFixed(2),⏎············})⏎·········`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5335813)

File: src/components/panels/EventsPanel.vue [hunk 3]
```diff
@@ -22,14 +22,14 @@
           <b-badge variant="danger">{{ event.victimNode }}</b-badge>
         </div>
         <b-badge v-if="isHealthReversed(event)" class="event-health" :variant="eventHealthVariantOpposite(event)">
-          {{ event.health || (100 - (event.currentScore / event.maximumScore) * 100).toFixed(2) }}% Completed
+          {{ $t('event.completed', {perc: event.health || (100 - (event.currentScore / event.maximumScore) * 100).toFixed(2) }) }}
```
Comment (EricSihaoLin) [inline src/components/panels/EventsPanel.vue:25]: Codacy Issue found: [Replace `·$t('event.completed',·{perc:·event.health·||·(100·-·(event.currentScore·/·event.maximumScore)·*·100).toFixed(2)·})` with `⏎············$t('event.completed',·{⏎··············perc:·event.health·||·(100·-·(event.currentScore·/·event.maximumScore)·*·100).toFixed(2),⏎············})⏎·········`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5335813)

File: src/components/NoDataItem.vue [hunk 4]
```diff
@@ -8,7 +8,7 @@
         height="40px"
         style="filter: invert(80%); padding-right:10px;"
       />
-      <div class="no-content-warning align-middle" style="margin-bottom:2px;">No {{ text }} Present</div>
+      <div class="no-content-warning align-middle" style="margin-bottom:2px;">{{ $t('nav.nodatatxt', { text: text }) }}</div>
```
Comment (EricSihaoLin) [inline src/components/NoDataItem.vue:11]: Codacy Issue found: [Replace `{{·$t('nav.nodatatxt',·{·text:·text·})·}}` with `⏎········{{·$t('nav.nodatatxt',·{·text:·text·})·}}⏎······`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5335813)

File: src/components/AsyncItemThumb.vue [hunk 5]
```diff
@@ -32,6 +32,10 @@ export default {
       type: Number,
       default: 20,
     },
+    ikey: {
+      type: String,
+      default: ''
+    }
```
Comment (EricSihaoLin) [inline src/components/AsyncItemThumb.vue:38]: Codacy Issue found: [Insert `,`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5335813)

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5335813&bid=17466094)

File: src/components/AsyncItemThumb.vue [hunk 6]
```diff
@@ -32,6 +32,10 @@ export default {
       type: Number,
       default: 20,
     },
+    ikey: {
+      type: String,
+      default: ''
```
Comment (EricSihaoLin) [inline src/components/AsyncItemThumb.vue:37]: Codacy Issue found: [Insert `,`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5335813)

**Comment by TobiTenno** [comment]
also fixes invasion images not showing up for non-english languages

**Review by EricSihaoLin — DISMISSED** [review]

**Review by EricSihaoLin — APPROVED** [review]

**Pull request merged**